### DISCUSSION TITLE
Feature Request - is making a transparent/blur taskbar option a good idea later in future instead of acylic? Happy with my windows now. Thanks

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#991] [Ideas]
![Screenshot 2022-03-04 222808](https://user-images.githubusercontent.com/100816109/156805149-03008140-221f-42d4-b357-01b93510e3bb.png)

**Comment by ghost** [comment]
Hi WiseTomCat,

I have been using the TranscluentTB app from the Windows store to have a transparent taskbar and its amazing, its free too :)

Shawn

**Reply by ghost** [reply]
> Hi WiseTomCat,
> 
> I have been using the TranscluentTB app from the Windows store to have a transparent taskbar and its amazing, its free too :)
> 
> Shawn

Hello there ghost, 
I tried the app long ago but is it stable with Explorer Patcher? The old windows 10 taskbar and buttons in windows 11 looks great with Explorer patcher. I think i will have to try myself how it works. Thanks

**Reply by Poopooracoocoo** [reply]
@WiseTomCat According to the [README](https://github.com/TranslucentTB/TranslucentTB/blob/release/README.md)
<img width="551" alt="image" src="https://user-images.githubusercontent.com/51363864/156880735-742610b5-f0f0-4b1e-8245-d42d4a8b6bd8.png">

**Reply by ghost** [reply]
thanks. it works really well with translucenttb. The taskbar color seems to change from dark to light and vice versa when switching apps and touching the taskbar stuffs but not a big deal for me. I just really wish it would be seamlessly implemented in here without having to use another app.

**Reply by Poopooracoocoo** [reply]
>I just really wish it would be seamlessly implemented in here without having to use another app.

This isn't the developer's job. I think it's too much to ask for duplicate functionality of other apps. ExplorerPatcher's main purpose is to bring back the Windows 10 taskbar, and now to bring back the Windows 10 start menu and other old flyouts. Other apps can then modify the Windows 10 taskbar.

And as you saw, the authors of popular apps that modify Explorer such as TranslucentTB, 7+ Taskbar Tweaker and more, recommend using ExplorerPatcher so they can modify the Windows 10 taskbar. There's really no point in duplicate work when it's all open source. 

>The taskbar color seems to change from dark to light and vice versa when switching apps and touching the taskbar stuffs but not a big deal for me

You might want to open an issue on the TranslucentTB repo.

**Reply by ghost** [reply]
thanks for the answer. My taskbar has very little or no acrylic effect like the original windows 10 taskbar as seen on my screenshot above so I tried to request that feature. I run windows on laptop mostly on battery so i want to use few apps as possible. Porting the start was a great success. It looks really good.

**Reply by valinet** [reply]
Thanks. Although, trust me, one more app doesn't really mean anything - I mean, what's the difference between something running standalone (TranslucentTB) or basically the same thing running in-process in `explorer.exe`. Yeah, maybe managed language vs C, although, is that really such a big deal nowadays...? I tend to put a lot of trust in compilers and JIT, improvements are constantly made in these areas.

**Reply by ghost** [reply]
If it would be a hassle to implement it then i understand but I faced few major bugs with Translucent TB on top of Explorer Patcher. The right task tray would move and get squeezed to the left side leaving the right side from the middle of taskbar empty so have to restart explorer everytime that happens. Switching between apps create slight flicker in taskbar. It would take 5-10 seconds after starting window for the Translucent TB to get applied and working. I use night and day theme with AutoDarkMode and the taskbar flickers a lot for 5 seconds and will be unusable for few seconds when themes get applied. So its pretty inconvenient as a daily driver. I was just implying that may be later in future if this transparency could get adopted then it would be a lot nicer.

**Reply by valinet** [reply]
If it is so bothersome in that app, then maybe it is just hard to do it right. I mean, yeah, the possibilities are richer when in process with the thing you’re trying to modify, but still. I don’t know. It’s a lot of work to get it done for what really? Default taskbar is fine imo, what’s that big of a difference between acrylic, blur, transparent (which is basically unusable most of the time)…? Yeah, I am not that eager to work in this area, maybe at some point in the future, idk. At the moment, version 42 already contains a ton of added features, I want to concentrate on polishing those and having them ship in a new release soon, rather than adding new functionality. Maybe TranslucentTB can enter some talks where they integrate their stuff with EP better, maybe take advantage of the fact EP is living in Explorer.exe, so it has access to more things (maybe more reliable notifications for various things etc).

**Reply by ghost** [reply]
hope it gets implemented someday. I was used to windows 10 looking so good and the start and taskbars were the few things windows implemented to have less customization so it would look on every pc out there. Right now Explorer Patcher on top of StartAllBack runs really beautifully but a tiny bit glitchy taskbar which is acceptable as long as it looks good. I also enjoy the mica being implemented on explorer.

**Comment by Shomnipotence** [comment]
It is a patch not an App, these commmand mean rebuild shell

**Reply by ghost** [reply]
> It is a patch not an App, these commmand mean rebuild shell

Yeah. My worry is that it might conlict with the Explorer Patcher taskbar since TranslucentTB has a lot of taskbar features. So, is it possible with explorer patcher may be later in future. Until a week ago i'd been using startallback but im sticking with explorer patcher.
